packages/cli/src/commands/test.js

```diff
@@ -0,0 +1,17 @@
+import execa from 'execa'
+
+import { getPaths } from 'src/lib'
+
+export const command = 'test'
+export const desc = 'Run Jest tests.'
+
+export const handler = () => {
+  //need to adjust import settings to be same as Webpack `directory-named-webpack-plugin`
+  execa('yarn jest --config=\'{"resolver":"jest-directory-named-resolver"}\'', {
+    cwd: getPaths().base,
+    shell: true,
+    stdio: 'inherit',
+  }).catch((e) => {
+    console.log(e)
+  })
```

Comment:
Nice! I think you should split this "jest" call into two separate commands, that way you can specify a config file for each environment. (Jest doesn't support overrides.)

The config files could live in `core/config/jest.api|web.js`

Reply:
working on this. a few things throwing me for a loop (or two). but I think I’m close.

Reply:
See new commit for dual cmds (api | web) and dual config files.